### ISSUE TITLE
Required `export ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.18`

### ISSUE THREAD

**Issue by zhilyaev** [helmwave/helmwave#309]
**General info (please complete the following information):**

- OS: all
- Helmwave Version 0.19.2

```
panic: Something in this program imports go4.org/unsafe/assume-no-moving-gc to declare that it assumes a non-moving garbage collector, but your version of go4.org/unsafe/assume-no-moving-gc hasn't been updated to assert that it's safe against the go1.18 runtime. If you want to risk it, run with environment variable ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.18 set. Notably, if go1.18 adds a moving garbage collector, this program is unsafe to use.

goroutine 1 [running]:
go4.org/unsafe/assume-no-moving-gc.init.0()

```

**Comment by oo00spy00oo** [comment]
The same issue with me

**Comment by zhilyaev** [comment]
This issue about go 1.18 runtime and go mod

```
go mod why go4.org/unsafe/assume-no-moving-gc
# go4.org/unsafe/assume-no-moving-gc
github.com/helmwave/helmwave/pkg/template
github.com/hairyhenderson/gomplate/v3
github.com/hairyhenderson/gomplate/v3/funcs
inet.af/netaddr
go4.org/intern
go4.org/unsafe/assume-no-moving-gc
```

**Comment by zhilyaev** [comment]
fix in v0.19.3
